server/svix-server/src/v1/endpoints/message.rs

```diff
@@ -182,25 +186,32 @@ pub struct MessageOut {
 }
 
 impl MessageOut {
-    pub fn without_payload(model: message::Model) -> Self {
+    pub fn from_msg_and_payload(model: message::Model, content: Option<Vec<u8>>) -> Self {
+        // Thankfully it can never fail :)
+        let payload = content
+            .map(|p| serde_json::from_slice(&p).expect("Can never fail"))
+            .or(model.legacy_payload);
+        let payload = RawPayload::from_string(match payload {
+            Some(payload) => serde_json::to_string(&payload).expect("Can never fail"),
+            None => r#"{"expired":true}"#.to_string(),
+        })
+        .expect("Can never fail");
```

Comment:
This is `MessageOut` so it's probably true, but last time I checked we had similar comments/remarks in the ingest handler when consuming `MessageIn` about "can never fail" even while json parsing _can fail_ when the structure is very deep, for example.

Reply:
Yeah, I honestly didn't feel comfortable w/ this and was hoping someone would call me out on it :stuck_out_tongue:. I still don't feel great about the `expect`s, but I went ahead and removed the most egregious one.

Reply:
More conservative is better here, yeah. I like your change. I'd normally only resort to the `expect()` if the containing scope forced me to make the body infallible, but that isn't the case here. 👍🏻